tests-system/rbt-quantified-expression/tracing

```diff
@@ -0,0 +1 @@
+LRM.Quantification_Type
```

Comment:
The good case of `Quantification_Type` is indeed tested with `quantified-expression`.
The bad case needs a test, too:
The predicate shall be something else, e.g. Integer:
```
checks T {
    (forall item in a => 42), warning "All items must be positive", a
}
```